ambry-store/src/main/java/com.github.ambry.store/BlobStore.java

```diff
@@ -139,8 +139,8 @@ public void start() throws StoreException {
         blobStoreStats =
             new BlobStoreStats(storeId, index, config.storeStatsBucketCount, config.storeStatsBucketSpanInMinutes,
```

Comment:
Good catch, I miscorrected when resolving the merge conflicts. Wait timeout is meant to be in seconds and BlobStoreStats also takes in as seconds.